util/cmake/SetCPackSystemName.cmake

```diff
@@ -44,8 +44,12 @@ if (CMAKE_SYSTEM_PROCESSOR STREQUAL "amd64")
   set(CMAKE_SYSTEM_PROCESSOR "x86_64")
 endif ()
 
-if ((CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64") AND (SEQAN_ARCH_SSE4))
-  set (CMAKE_SYSTEM_PROCESSOR "x86_64_sse4") 
+if (CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" AND (SEQAN_ARCH_SSE4))
+  if (SEQAN_ARCH_AVX2)
+    set (CMAKE_SYSTEM_PROCESSOR "x86_64_avx2") 
+  else ()
+    set (CMAKE_SYSTEM_PROCESSOR "x86_64_sse4") 
+  endif ()
```

Comment:
@h-2 
I made this according to https://github.com/seqan/seqan/blob/develop/util/cmake/SeqAnBuildSystem.cmake#L287
However, correct me if it's not what you wanted. :)